nodejs/week1/homework/warmup.js

```diff
@@ -0,0 +1,22 @@
+console.log("inside warmup file");
+
+class Circle {
+    constructor(radius){
+        this.radius = radius
+    }
+    getRadius(){
+        return this.radius * 2
+    }
+    getCircumference(){
+        return 2 * 3,14 * this.radius
```

Comment:
Instead of 3,14, you could use Math.PI, which is going to give a more precise result.